app/javascript/widget/components/dropdown/DropdownMenuItem.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <button :class="['menu-item', itemClass]" @click="action">
+    <fluent-icon
+      v-if="icon"
+      :icon="iconName"
+      :size="iconSize"
+      :class="iconClass"
+    />
+    <span :class="[{ 'pl-3': icon }, textClass]">{{ text }}</span>
+  </button>
+</template>
+<script>
+import FluentIcon from 'shared/components/FluentIcon/Index.vue';
+
+export default {
+  components: { FluentIcon },
+  props: {
+    text: {
+      type: String,
+      default: 'Default',
+    },
+    textClass: {
+      type: String,
+      default: 'text-sm',
+    },
+    icon: {
+      type: Boolean,
+      default: true,
+    },
+    iconName: {
+      type: String,
+      default: '',
+    },
+    iconSize: {
+      type: String,
+      default: '15',
+    },
+    iconClass: {
+      type: String,
+      default: 'text-black-900',
+    },
+    itemClass: {
+      type: String,
+      default: 'flex items-center p-3 cursor-pointer ml-0',
+    },
+    action: {
+      type: Function,
+      default: () => {},
+    },
+  },
+};
+</script>
+<style scoped lang="scss">
+@import '~widget/assets/scss/variables.scss';
+
+.menu-item {
+  border-bottom: 1.5px solid $color-border;
```

Comment:
1.5px seems off. Any particular reason to choose the measurement?

Reply:
Updated to tailwind classes